### ISSUE TITLE
Eclipse plugin installation fails

### ISSUE THREAD

**Issue by markusschuh** [devonfw/devon-ide#65]
Category: Bug

Severity: Low

Expected behaviour "./setup" should succeed without warnings about failed eclipse plugin installation

Actual/current behaviour: "./setup" outputs:
>    ******** ATTENTION ********
>    WARNING: Installation of plugin AnyEditTools failed. Please try to install manually.
>    Installing eclipse plugin net.sf.eclipsecs from http://eclipse-cs.sf.net/update/
>    No Java runtime present, requesting install. 

Steps to reproduce:  Remove all devon-ide introduced changes on system and run "./setup" inside devon-ide root directory again.

Comments: On macOS, Eclipse may need an additional "vm" option - either on command line or set via eclipse.ini. http://wiki.eclipse.org/Eclipse.ini#-vm_value:_Mac_OS_X_Example
I assume, the plugin install start of eclipse could just use the same additional option as the later executed eclipse start:
 -vm "${JAVA_HOME}/bin/java"

macOS 10.14.4, devon-ide-scripts-3.0.0-beta2.tar.gz (from Maven)

**Comment by hohwille** [comment]
Thanks for the feedback and the hint. I will add the `vm` option and also the `vmargs` options (as I have seen OOME once in a while) to the command for the plugin installation.

**Comment by hohwille** [comment]
I could resolve the above error but still get this one:
```
Installing org.eclipse.eclemma.feature.feature.group 3.1.2.201903112331.
Installation failed.
Cannot complete the install because of a conflicting dependency.
	Software being installed: EclEmma Java Code Coverage 3.1.2.201903112331 (org.eclipse.eclemma.feature.feature.group 3.1.2.201903112331)
	Software currently installed: EclEmma Java Code Coverage 3.1.1.201809121651 (org.eclipse.eclemma.feature.feature.group 3.1.1.201809121651)
	Only one of the following can be installed at once:
		Old EclEmma version (com.mountainminds.eclemma) cannot be installed together with new (org.eclipse.eclemma). Uninstall currently installed version and try the install again. 3.1.2.201903112331 (com.mountainminds.eclemma.core 3.1.2.201903112331)
		Old EclEmma version (com.mountainminds.eclemma) cannot be installed together with new (org.eclipse.eclemma). Uninstall currently installed version and try the install again. 3.1.1.201809121651 (com.mountainminds.eclemma.core 3.1.1.201809121651)
	Cannot satisfy dependency:
		From: EclEmma Java Code Coverage 3.1.1.201809121651 (org.eclipse.eclemma.feature.feature.group 3.1.1.201809121651)
		To: osgi.bundle; com.mountainminds.eclemma.core [3.1.1.201809121651,3.1.1.201809121651]
	Cannot satisfy dependency:
		From: EclEmma Java Code Coverage 3.1.2.201903112331 (org.eclipse.eclemma.feature.feature.group 3.1.2.201903112331)
		To: osgi.bundle; com.mountainminds.eclemma.core [3.1.2.201903112331,3.1.2.201903112331]
There were errors. See log file: /projects/ide-test/software/eclipse/Eclipse.app/Contents/Eclipse/configuration/1554130633195.log
```

**Comment by hohwille** [comment]
Seems as if EclEmma is already part of the official Eclipse vanilla distribution and therefore we can ommit it in our plugin list. Anyhow Eclipse should perform an update as the plugin is outdated but lets forget about this detail for the moment to proceed...

**Comment by hohwille** [comment]
Also I see this error:
```
Installing org.sonar.ide.eclipse.feature.feature.group 3.5.0.20150804-1512-RELEASE.
Installing org.sonar.ide.eclipse.jdt.feature.feature.group 3.5.0.20150804-1512-RELEASE.
Installation failed.
Cannot complete the install because one or more required items could not be found.
	Software being installed: SonarQube Java Configuration Helper 3.5.0.20150804-1512-RELEASE (org.sonar.ide.eclipse.jdt.feature.feature.group 3.5.0.20150804-1512-RELEASE)
	Missing requirement: Apache Apache HttpClient OSGi bundle 4.3.2 (org.apache.httpcomponents.httpclient 4.3.2) requires 'java.package; org.apache.commons.logging [1.1.0,1.2.0)' but it could not be found
	Cannot satisfy dependency:
		From: SonarQube Integration for Eclipse 3.5.0.20150804-1512-RELEASE (org.sonar.ide.eclipse.feature.feature.group 3.5.0.20150804-1512-RELEASE)
		To: org.eclipse.equinox.p2.iu; org.apache.httpcomponents.httpclient [4.3.2,4.3.2]
	Cannot satisfy dependency:
		From: SonarQube Java Configuration Helper 3.5.0.20150804-1512-RELEASE (org.sonar.ide.eclipse.jdt.feature.feature.group 3.5.0.20150804-1512-RELEASE)
		To: org.eclipse.equinox.p2.iu; org.sonar.ide.eclipse.feature.feature.group 0.0.0
There were errors. See log file: /projects/ide-test/software/eclipse/Eclipse.app/Contents/Eclipse/configuration/1554130748509.log
```

**Comment by hohwille** [comment]
Seems that sonarlint has additinal dependencies to other plugins. IMHO sonarlint is too greedy anyhow and massively slows down the IDE so it should not come as default. Still it would be good to have a way for projects that want to have it to be able to install it automatically via their project specific git config (settings).

**Comment by hohwille** [comment]
Fixed and PR #71 merged.
